eng/Directory.Build.Data.targets

```diff
@@ -28,17 +28,13 @@
     <PropertyGroup>
       <PowerShellExe Condition=" '$(PowerShellExe)'=='' ">"%ProgramFiles%\PowerShell\6\pwsh.exe"</PowerShellExe>
       <PowerShellExe Condition="!Exists('$(PowerShellExe)')">pwsh</PowerShellExe>
-      <GetReleaseNotesScriptPath Condition=" '$(GetReleaseNotesScriptPath)'=='' ">$(MSBuildThisFileDirectory)/common/Extract-ReleaseNotes.ps1</GetReleaseNotesScriptPath>
+      <ValidateReleaseNotesScriptPath Condition=" '$(ValidateReleaseNotesScriptPath)'=='' ">$(MSBuildThisFileDirectory)common/scripts/Verify-ChangeLog.ps1</ValidateReleaseNotesScriptPath>
       <ChangeLogPath>$([MSBuild]::NormalizeDirectory($(MSBuildProjectDirectory)/../))CHANGELOG.md</ChangeLogPath>
     </PropertyGroup>
-    <Exec Condition="Exists('$(ChangeLogPath)')" ContinueOnError="true" ConsoleToMSBuild="true" Command="$(PowerShellExe) -NonInteractive -executionpolicy Unrestricted -File $(GetReleaseNotesScriptPath) $(ChangeLogPath) $(_VersionInProject)">
-      <Output TaskParameter="ConsoleOutput" ItemName="ExtractedReleaseNotesTemp" />
+    <Exec Condition="Exists('$(ChangeLogPath)')" ContinueOnError="true" ConsoleToMSBuild="true" Command="$(PowerShellExe) -NonInteractive -executionpolicy Unrestricted -File $(ValidateReleaseNotesScriptPath) $(ChangeLogPath) $(_VersionInProject)">
```

Comment:
I suggest using the command line parameter names for the script so we can ensure they get set for the correct parameters.

Reply:
I still think  this PR should be merged first, then it can be Overrode on the next eng/common sync.  Merging this changes in eng/common first will break the pipeline here.

Reply:
I'll leave the order up to you, you understand the risks if something else gets in and merged into eng/common before your change it could break this but as long as you watch them closely you can do whatever order you like.